jamventProto/jamctrllib/JamCtrlHw.cpp

```diff
@@ -0,0 +1,28 @@
+
+#include "JamCtrlHw.h"
+#include "fmtstr.h"
```

Comment:
Wondering if we really plan to use this.  Assume it is printf functionality for logging.  I've been using qDebug() for logging in the GUI Qt code.  See other logging inserting (<<) to a stream.  Guess we should decide this as a project.